### PR TITLE
HPCC-16565 ws_access/UserQuery needs to return employeeID

### PR DESCRIPTION
**Pull request by RussWhitehead**
Employee ID not returned from ws_access UserQuery feature. This PR adds
it to the returned values

Signed-off-by: Russ Whitehead <william.whitehead@lexisnexis.com>

### CODE REVIEW

**Comment by RussWhitehead** [comment]
@rpastrana Please review

File: esp/scm/ws_access.ecm [hunk 1]
```diff
@@ -20,6 +20,7 @@ ESPenum UserSortBy : string
     Name("Name"),
     FullName("FullName"),
     PasswordExpiration("PasswordExpiration"),
+    EmployeeID("EmployeeID"),
```
Comment (rpastrana) [inline esp/scm/ws_access.ecm]: should this be versioned?

File: system/security/LdapSecurity/ldapconnection.cpp [hunk 2]
```diff
@@ -2533,6 +2533,7 @@ class CLdapClient : implements ILdapClient, public CInterface
             userTree->addProp(getUserFieldNames(UFFullName), fullName);
```
Comment (rpastrana) [inline system/security/LdapSecurity/ldapconnection.cpp]: legacy, but worth fixing the indentation issue here (fixing it won't confuse reviewer)

**Review by rpastrana — CHANGES_REQUESTED** [review]
2 minor comments.

**Comment by RussWhitehead** [comment]
@ghalliday Please merge

**Comment by miguelvazq** [comment]
Can we make sure that the FilterBy options are as follows:

username vs. Name
fullname vs. FullName
passwordexpiration vs. PasswordExpiration
employeeID vs. EmployeeID

**Pull request merged**